securitycenter/noxfile.py

```diff
@@ -84,6 +84,7 @@ def cover(session):
     """
     session.chdir(os.path.dirname(__file__))
     session.install('coverage', 'pytest-cov')
-    session.run('coverage', 'report', '--show-missing', '--fail-under=100')
+    # TODO: fail-under should be changed to 100.
```

Comment:
Create an issue for it?  I don't know why the generated unit test would have less than 100% coverage.

Reply:
https://github.com/googleapis/gapic-generator/issues/2398